src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -981,33 +988,34 @@ It is also possible to change the ProActive Nodes Account using the 'Change Acco
 
 When you click on 'GUI Edit', the following window appears:
 
-image::config_editor_general.png[align="center"]
+image::config_editor_general.png[align=center, width=612, height=306]
 
 In the general tab, the user can specify:
 
 * The *ProActive Scheduler* location.
-* The *JRE* location (usually something like `C:\Program Files\Java\jdk1.6.0_12`).
+* The *JRE* location (the Java Runtime Environment is provided with the agent archive under this path `C:\Program Files\ProActive Agent\schedworker\jre`).
```

Comment:
Last remark: as bellow we are using the environement variable while indicating path, maybe, it would be better to mention `%programfiles(x86)%\ProActive Agent\schedworker\jre` instead of `C:\Program Files\ProActive Agent\schedworker\jre`